providers/os/connection/docker/docker_file_connection.go

```diff
@@ -86,7 +90,7 @@ func NewDockerfileConnection(id uint32, conf *inventory.Config, asset *inventory
 
 	conn := &DockerfileConnection{
 		LocalConnection: localConn,
-		Filename:        filename,
+		Filename:        filepath.Join(conf.Path, filename),
```

Comment:
Why can't we use `absSrc` directly?

Reply:
because `absSrc` gets overwritten to only the directory, not the full path.